pages/urlEncodeDecode.tsx

```diff
@@ -109,21 +116,82 @@ export default function URLEncodeDecode() {
             )}
           </Box>
         </Box>
+
         <TextField
           label='Encoded'
-          data-testid='Encoded'
+          data-testid='Output - Encoded'
           value={encoded}
           onChange={(e) => setEncoded(e.target.value)}
           multiline
         />
+        <Box
+          display='flex'
+          flexWrap='wrap'
+          justifyContent='end'
+          gap={2}
```

Comment:
I don't think we need this gap here, it makes the inputs on the page not uniform because the first input doesn't have a gap.